tardis/visualization/tools/sdec_plot.py

```diff
@@ -514,15 +519,74 @@ def _calculate_plotting_data(
         # Calculate luminosities to be shown in plot
         (
             self.emission_luminosities_df,
-            self.elements,
+            self.emission_elements,
         ) = self._calculate_emission_luminosities(
             packets_mode=packets_mode, packet_wvl_range=packet_wvl_range
         )
-        self.absorption_luminosities_df = (
-            self._calculate_absorption_luminosities(
-                packets_mode=packets_mode, packet_wvl_range=packet_wvl_range
-            )
+        (
+            self.absorption_luminosities_df,
+            self.absorption_elements,
+        ) = self._calculate_absorption_luminosities(
+            packets_mode=packets_mode, packet_wvl_range=packet_wvl_range
+        )
+
+        # Calculate the total contribution of elements
+        # by summing absorption and emission
+        self.total_luminosities_df = (
+            self.absorption_luminosities_df + self.emission_luminosities_df
+        )
+
+        # Drop no interaction and electron scattering so you can sort element contributions
+        self.total_luminosities_df.drop(
+            ["noint", "escatter"], axis=1, inplace=True
+        )
```

Comment:
How about dropping noint & escatter from emission_luminosities_df before summing? 

IMO this will save us from adding these two columns with all values zero in absorption_luminosities_df (i.e. line 800-801) just for sake of summing here.

Reply:
Done